### PR TITLE
feat: add bun support

### PR DESCRIPTION
**Pull request by DjDeveloperr**

### CODE REVIEW

**Review by eliassjogreen — APPROVED** [review]
I think it looks good for a first iteration on bun support! Great work!

File: plugin.ts
Comment (eliassjogreen) [inline plugin.ts:1]: For bun, a dts file with
```ts
declare module "python:*";
declare module "*.py";
```
probably also makes sense.

**Review by eliassjogreen — CHANGES_REQUESTED** [review]

File: plugin.ts
Comment (DjDeveloperr) [reply plugin.ts:1]: Hmm, but if we add this, Deno will also consider these as valid imports while Bun will not, is that right?

**Review by eliassjogreen — APPROVED** [review]
Nice! Good job :D

**Pull request merged**